docs/design/code_and_name_organization/README.md

```diff
@@ -272,28 +370,45 @@ text. `PACKAGE//default` will refer to the name of the library used when no
 `library` argument is specified, although `PACKAGE` may also be used in
 situations where it is unambiguous that it still refers to the default library.
 
+The name `Main` is reserved as a package name and cannot appear within source
```

Comment:
What's the reason for reserving it?

Reply:
I believe this has been addressed by the choice to give the main package an actual name of `Main`.